frontend/pdc/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "pdc",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "prepare": "cd ../.. && husky install frontend/pdc/.husky"
+  },
+  "dependencies": {
+    "next": "10.2.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
+  },
+  "devDependencies": {
+    "@types/react": "^17.0.5",
+    "husky": "^6.0.0",
+    "typescript": "^4.2.4"
```

Comment:
no need for typescript we will have testing for sure. Also no need for types react